core/src/main/scala/caliban/wrappers/ApolloPersistedQueries.scala

```diff
@@ -6,6 +6,9 @@ import caliban.wrappers.Wrapper.OverallWrapper
 import caliban.{ CalibanError, GraphQLRequest, GraphQLResponse, InputValue }
 import zio.{ Has, Layer, Ref, UIO, ZIO }
 
+import java.nio.charset.StandardCharsets
+import javax.xml.bind.DatatypeConverter
```

Comment:
Hmm it doesn't find this in the CI, is that specific to some JVMs?

Reply:
Fixed, I thought javax.xml would be safe but it wasn't too hard to write my own.